core/layers/svc-core-proxy/extension/clients/elasticsearch_client.go

```
@@ -0,0 +1,106 @@
+	return client, nil
+}
+
+func (c *ElasticSearchClient) FetchAssetDetails(ctx context.Context, tenantId, ag, assetId string, size int) (*map[string]interface{}, error) {
+	query := buildQuery(assetId)
+	esRequest := map[string]interface{}{
+		"size":  size,
+		"query": query,
+	}
+
+	var buffer bytes.Buffer
+	json.NewEncoder(&buffer).Encode(esRequest)
```

golangci-lint (1.62.2)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)